src/output/table.rs

```diff
@@ -116,12 +124,18 @@ pub enum Column {
     Permissions,
     FileSize,
     Timestamp(TimeType),
+    #[cfg(unix)]
     Blocks,
+    #[cfg(unix)]
     User,
+    #[cfg(unix)]
     Group,
+    #[cfg(unix)]
```

Comment:
Windows [has hard links](https://docs.microsoft.com/en-us/windows/win32/fileio/hard-links-and-junctions) too